db-engine/src/main/java/com/splicemachine/db/impl/sql/GenericStatement.java

```diff
@@ -725,9 +725,14 @@ private void dumpOptimizedTree(LanguageConnectionContext lcc,StatementNode qt,bo
      */
     private void walkAST(LanguageConnectionContext lcc, Visitable queryTree, CompilationPhase phase) throws StandardException {
         ASTVisitor visitor = lcc.getASTVisitor();
-        if (visitor != null) {
-            visitor.begin(statementText, phase);
-            queryTree.accept(visitor);
+        try {
+            if (visitor != null) {
+                visitor.begin(statementText, phase);
+                queryTree.accept(visitor);
+            }
+        } catch (StandardException e) {
```

Comment:
You don't need the catch clause, unless you want to be explicit.

Reply:
Thanks @dgomezferro . I've updated the PR to remove the catch clause.